source/POI.DiscordDotNet/POI.DiscordDotNet.csproj

```diff
@@ -9,11 +9,10 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="DSharpPlus" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.CommandsNext" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.Interactivity" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.Rest" Version="4.4.2" />
-        <PackageReference Include="DSharpPlus.SlashCommands" Version="4.4.2" />
+        <PackageReference Include="DSharpPlus" Version="5.0.0-nightly-01663" />
```

Comment:
Did you check all major changes?

Reply:
Testing in production.

Reply:
> Testing in production.

![afbeelding](https://github.com/ErisApps/POI/assets/36133967/8d8b6c04-97c3-470a-add0-6bea9f1793a8)

Reply:
![770049584222699570](https://github.com/ErisApps/POI/assets/25928757/3ddcc8ee-5c3f-4dcd-82a5-a08152f1247c)